EngineLayer/FdrAnalysis/FdrAnalysisEngine.cs

```diff
@@ -281,13 +281,29 @@ private static int GetNumPSMsAtqValueCutoff(List<PeptideSpectralMatch> psms, dou
         public void CountPsm()
         {
             // exclude ambiguous psms and has a fdr cutoff = 0.01
-            var psmsGroupedBySequence = AllPsms.Where(psm => psm.FullSequence != null && psm.FdrInfo.QValue <= 0.01 && psm.FdrInfo.QValueNotch <= 0.01)
-                .GroupBy(p => p.FullSequence).ToList();
+            var allUnambiguousPsms = AllPsms.Where(psm => psm.FullSequence != null);
 
-            foreach (var group in psmsGroupedBySequence)
+            var unambiguousPsmsLessThanOnePercentFdr = allUnambiguousPsms.Where(psm =>
```

Comment:
What fraction of PSMs are ambiguous in a normal search?

Reply:
![image](https://user-images.githubusercontent.com/16841846/58652990-60cc6480-82da-11e9-8ecd-c6b46a218842.png)

Reply:
That's an interesting way to look at this. It's an exponential curve of sorts.

Thinking of this as a histogram, it's probably a pretty similar fraction of PSMs that are ambiguous as what I see in variant searches. ~5 out of every 6 PSMs

Reply:
Do you have that plot before/after your p-value stuff?

Reply:
this is psm level 
![image](https://user-images.githubusercontent.com/16841846/58656650-52824680-82e2-11e9-8142-657033dd2603.png)

Reply:
this is accession level
![image](https://user-images.githubusercontent.com/16841846/58656705-629a2600-82e2-11e9-8421-d81e45a84c41.png)

Reply:
20% more with a single accession after p-value filtering

Reply:
That's awesome. Is this new, or is this building off a published method?

Reply:
Could you plot the fractions? `(filtered - unfiltered) / unfiltered`

Reply:
"new" I using a new machine learning package from microsoft that's compatible with .net core. This one does binary classification (Target or Decoy) with probability based on ~8 factors

Reply:
that excel sheet is long gone. when I get back from ASMS